app/src/main/kotlin/com/wire/android/ui/home/conversations/edit/EditMessageMenuItems.kt

```diff
@@ -1,22 +1,80 @@
 import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.LocalMinimumTouchTargetEnforcement
 import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.wire.android.R
 import com.wire.android.ui.common.bottomsheet.MenuBottomSheetItem
 import com.wire.android.ui.common.bottomsheet.MenuItemIcon
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireTypography
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun EditMessageMenuItems(
     isCopyable: Boolean,
     isEditable: Boolean,
     onCopyMessage: () -> Unit,
     onDeleteMessage: () -> Unit,
+    onReactionClick: (emoji: String) -> Unit,
 ): List<@Composable () -> Unit> {
     return buildList {
+        add {
+            Column() {
+                Row() {
+                    Spacer(modifier = Modifier.width(8.dp))
+                    Text(stringResource(R.string.label_reactions).uppercase(), style = MaterialTheme.wireTypography.label01)
+                }
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    listOf("👍", "❤️", "🚀", "🤯", "😄", "🤣", "👎").forEach { emoji ->
+                        CompositionLocalProvider(
+                            LocalMinimumTouchTargetEnforcement provides false
+                        ) {
+                            Button(
+                                onClick = {
+                                    // So we display the pretty emoji,
+                                    // but we match the ugly one sent from other platforms
+                                    val correctedEmoji = if (emoji == "❤️") "❤"
+                                    else emoji
+                                    onReactionClick(correctedEmoji)
+                                },
+                                modifier = Modifier.defaultMinSize(minWidth = 1.dp, minHeight = 1.dp),
+                                contentPadding = PaddingValues(8.dp),
+                                colors = ButtonDefaults.buttonColors(
+                                    containerColor = MaterialTheme.wireColorScheme.surface,
+                                    contentColor = MaterialTheme.wireColorScheme.secondaryButtonSelectedOutline
+                                )
+                            ) {
+                                Text(emoji, style = TextStyle(fontSize = 28.sp))
+                            }
+                        }
+                    }
+                }
+            }
```

Comment:
I would suggest to move this to a separate Composable function, something like `MessageReactions()` ?, in sake of not making `EditMessageMenuItems` a big function